pkg/proxy/api/endpoints.go

```diff
@@ -142,6 +144,23 @@ func (ep *endpointsProcessor) setStateMultiple(c *gin.Context) {
 	shared.RespondWith(c, http.StatusOK, gin.H{}, "", data.ReturnCodeSuccess)
 }
 
+func (ep *endpointsProcessor) setStateMultipleOverwrite(c *gin.Context) {
+	var stateSlice []*dtos.AddressState
+	err := c.ShouldBindJSON(&stateSlice)
+	if err != nil {
+		shared.RespondWithBadRequest(c, fmt.Sprintf("invalid state structure, error: %s", err.Error()))
+		return
+	}
+
+	err = ep.facade.SetStateMultipleOverwrite(stateSlice)
+	if err != nil {
+		shared.RespondWithBadRequest(c, fmt.Sprintf("cannot set state, error: %s", err.Error()))
```

Comment:
```suggestion
		shared.RespondWithBadRequest(c, fmt.Sprintf("cannot overwrite state, error: %s", err.Error()))
```
or something similar to defferentiate from simple set state

Reply:
done